TestHostApp build fails in VS2022 ('/ZW' and '/std:c++20' command-line options are incompatible)

### Windows Terminal version

f17c2babd4f22c21b9ad0897f884b2b3dbe173f1

### Windows build number

_No response_

### Other Software

Visual Studio 2022 (17.1.1)

### Steps to reproduce

Clone repo
Restore git submodules
Open OpenConsole.sln in Visual Studio
<work around https://github.com/microsoft/terminal/issues/12671>
Build project x64|Debug

### Expected Behavior

The build succeeds

### Actual Behavior

`4>------ Build started: Project: TestHostApp, Configuration: Debug x64 ------
4>main
4>C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(560,5): error MSB6006: "CL.exe" exited with code 2.
4>cl : command line error D8016: '/ZW' and '/std:c++20' command-line options are incompatible
4>Done building project "TestHostApp.vcxproj" -- FAILED.`

--- a/src/cascadia/LocalTests_TerminalApp/TestHostApp/TestHostApp.vcxproj
+++ b/src/cascadia/LocalTests_TerminalApp/TestHostApp/TestHostApp.vcxproj
@@ -51,7 +51,8 @@
 
   <ItemDefinitionGroup>
     <ClCompile>
-      <!-- Disable two-phase name lookup for C++/CLI & C++/CX code. -->
+      <!-- Disable C++20 and two-phase name lookup for C++/CLI & C++/CX code. -->
+      <LanguageStandard>stdcpp17</LanguageStandard>
       <AdditionalOptions>%(AdditionalOptions) /Zc:twoPhase-</AdditionalOptions>
       <DisableSpecificWarnings>4453;%(DisableSpecificWarnings)</DisableSpecificWarnings>
       <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(IntermediateOutputPath)</AdditionalIncludeDirectories>

